docs/tutorial.rst

```diff
@@ -204,6 +204,47 @@ here is an array using Zstandard compression, level 1::
     >>> z.compressor
     Zstd(level=1)
 
+Getting the diagnostics on the compression ratio :
+
+    >>> z.info
+    Type               : zarr.core.Array
+    Data type          : int32
+    Shape              : (10000, 10000)
+    Chunk shape        : (1000, 1000)
+    Order              : C
+    Read-only          : False
+    Compressor         : Zstd(level=1)
+    Store type         : zarr.storage.KVStore
+    No. bytes          : 400000000 (381.5M)
+    No. bytes stored   : 299348530 (285.5M)
+    Storage ratio      : 1.3
+    Chunks initialized : 100/100
+
+
+Here is also an array using GZip compression, level 2::
+
+    >>> from numcodecs import GZip
+    >>> z = zarr.array(np.arange(100000000, dtype='i4').reshape(10000, 10000),
+    ...                chunks=(1000, 1000), compressor=GZip(level=2))
+    >>> z.compressor
+    GZip(level=2)
```

Comment:
This too.

Reply:
Sorry that was my mistake @MSanKeys963.
I would work on that right away!

Reply:
Hello @MSanKeys963 . I have resolved the issue